.github/workflows/pr-scope-check.yml

```
@@ -0,0 +1,65 @@
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+
+    - name: Run PR Scope Check
+      run: |
+        python scripts/check_pr_scope.py --strict
+      continue-on-error: false
+
+    - name: Check branch naming
+      run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)